packages/contracts/contracts/lib/LibAsset.sol

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: UNLICENSED
-pragma solidity 0.8.4;
+pragma solidity >=0.7.6;
 
-import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "./OVM_SafeERC20.sol";
```

Comment:
Where is this contract? Same with the `Address` one..?

What's the diff between `SafeERC20` and `OVM_SafeERC20`?

Reply:
Added